src/app/(tabs)/search/index.tsx

```diff
@@ -222,19 +223,31 @@ function SearchScreen() {
         )}
 
         {showRecents &&
-          (search ? (
+          (search && searchResults.length > 0 ? (
             <View style={styles.default}>
               <Text style={[styles.searchText, styles.numDisplay]}>
                 {searchResults.length}{' '}
                 {searchResults.length === 1 ? 'Story' : 'Stories'}
               </Text>
             </View>
-          ) : (
+          ) : search && searchResults.length === 0 ? (
+            <View style={styles.emptySearch}>
+              <View style={{ paddingBottom: 16 }}>
+                <Text style={globalStyles.h3}>There are no stories</Text>
```

Comment:
I think you looked at the filter page no results screen instead of the search page no results screen. This is the design for the no results screen:
<img width="372" alt="Screenshot 2024-03-26 at 8 16 15 PM" src="https://github.com/calblueprint/girls-write-now/assets/34043950/889c8adf-8222-414c-a8a7-471330edf4f3">